index.bs

```diff
@@ -402,6 +405,10 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
             If {{VideoConfiguration/framerate}} is not finite or is not greater
             than 0, return <code>false</code> and abort these steps.
           </li>
+          <li>
+            If an optional member is specified for a {{MediaDecodingType}} or
+            {{MediaEncodingType}} to which it's not applicable, return
+            <code>false</code> and abort these steps.
```

Comment:
Append: See applicability rules in the member definitions below.

Reply:
Done!